drivers/video/video_mcux_csi.c

```diff
@@ -101,10 +103,13 @@ static void __frame_done_cb(CSI_Type *base, csi_handle_t *handle,
 	k_fifo_put(&data->fifo_out, vbuf);
 
 done:
+#ifdef CONFIG_POLL
```

Comment:
You can use ` if (IS_ENABLED(CONFIG_POLL))`

Reply:
ack, changed.

Reply:
indeed, thanks, done.